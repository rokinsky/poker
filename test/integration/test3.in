omaha-holdem KhQh4hAc3h AsQdQsJd Jh8s7d6h Ah8h6s4c
omaha-holdem 4sQhKh3sAc AhQdTcTh KdQcQs4d JhTs7c4c
omaha-holdem Th2sQcQd4s KhQhJh7c Kc6c6d2c
omaha-holdem 2hJcJh6s2d KhTd4c4h AcAdTh8c
omaha-holdem 8c3sAdAh5s Ts9s8s6d AsQsTh3h
omaha-holdem Jc9cKs4s3d QdQsJh7h KcTs7s2c
omaha-holdem KcJcKs3h5c AcKh9h2c Kd5d5h3c
omaha-holdem 5d4dKs2s3d KhQdQs4s Ac6h3c2d
omaha-holdem 6cJs3h5dTh AcAh9c9h AsKhQhJh 6h4c3c2h
omaha-holdem 2hKd8s5hKs AcAh6c4c AsKcKhJd
omaha-holdem Jd3cTh9hAc QhTs9d7h KhTcTd9s
omaha-holdem ThJdAh8c2c JcJs5d5s AdAs9c6c KcKsQh8h
omaha-holdem 7s7dKsKhQs 2c2d3c3d 4c4d5c5d 6c6d7c7h
omaha-holdem 3d3s4d6hJc Js2dKd8c KsAsTcTs Jh2h3c9c Qc8dAd6c 7dQsAc5d
omaha-holdem QdTd3sJcTs 4d3h2hQh 4c8s9d5d
omaha-holdem KhQdQsKs5d KcQcJd7d AcAs5h2c
omaha-holdem 4h7s4d2h5c KdKs7h5d AhJc5h4c
omaha-holdem 4s5cQc3hQh AsJcJs3c Td9s8d4c
omaha-holdem 6c4s7c9c9d 8h7d5s2d AcKhTs8c
omaha-holdem 9c8cKh3sQs AsKcKs9d KdTdTs2s
omaha-holdem 6c4s7c9c9d 8h7d5s2d AcKhTs8c
omaha-holdem 7h8dKcTh9h AhAsJs6h AcKhTc2d
omaha-holdem 4h6s3dAhQh Ac7c5c2c Kh6h3h3s
omaha-holdem 4c3s9c2d6h 6c5d4d2c Tc9d9h3h
omaha-holdem AhTc9c4sQh Ad8h7c4d AsTs8d2h
omaha-holdem 9sQd7c8c8s KcKhQsJh Ad9c7s4d
omaha-holdem Tc8sQs6d2s JcTh9h6s JsTs9s7h As7d5s2d
omaha-holdem 3dJd8d8sAs Ac7c4h3h AdAhQd6d
omaha-holdem 4hJdTh3s8c AhJh9h6d QdTdTs2h Kh8d7s6h
omaha-holdem Qs2cKs8c6h JhTh7s6s QdQh4d3d
omaha-holdem 5dQs9h8s4c KdQd9c5c JdJs9d6s
