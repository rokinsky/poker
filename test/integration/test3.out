AsQdQsJd Jh8s7d6h Ah8h6s4c
AhQdTcTh KdQcQs4d JhTs7c4c
Kc6c6d2c KhQhJh7c
KhTd4c4h AcAdTh8c
Ts9s8s6d AsQsTh3h
QdQsJh7h KcTs7s2c
AcKh9h2c Kd5d5h3c
KhQdQs4s Ac6h3c2d
AsKhQhJh AcAh9c9h 6h4c3c2h
AcAh6c4c AsKcKhJd
QhTs9d7h KhTcTd9s
JcJs5d5s AdAs9c6c KcKsQh8h
2c2d3c3d 4c4d5c5d 6c6d7c7h
Qc8dAd6c KsAsTcTs Js2dKd8c 7dQsAc5d Jh2h3c9c
4d3h2hQh 4c8s9d5d
AcAs5h2c KcQcJd7d
KdKs7h5d AhJc5h4c
Td9s8d4c AsJcJs3c
8h7d5s2d AcKhTs8c
KdTdTs2s AsKcKs9d
8h7d5s2d AcKhTs8c
AcKhTc2d AhAsJs6h
Ac7c5c2c Kh6h3h3s
Tc9d9h3h 6c5d4d2c
Ad8h7c4d AsTs8d2h
Ad9c7s4d KcKhQsJh
JcTh9h6s JsTs9s7h As7d5s2d
Ac7c4h3h AdAhQd6d
Kh8d7s6h AhJh9h6d QdTdTs2h
JhTh7s6s QdQh4d3d
JdJs9d6s KdQd9c5c
